### PR TITLE
Added Home page & Categories

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Home page & Categories

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Excellent code structure, Everything looks fine. Good to go from my side.

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import url(/styles/utility.css);
+@import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css);
+
+.img-size {
```
Comment (FarhanMobashir) [inline styles/index.css]: name your classes in a more meaningful way so that when you come back to your code it will be easy to refactor and catch bugs

**Review by FarhanMobashir — APPROVED** [review]
left some suggestion

File: styles/index.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/index.css]: Thanks for the comment, changed the class name

**Pull request merged**